test/models/site_test.rb

```diff
@@ -61,4 +61,35 @@ class SiteTest < ActiveSupport::TestCase
       assert_equal(["はすでに存在します"], site.errors[:fqdn])
     end
   end
+
+  sub_test_case "category_title_format" do
+    test "null" do
+      site = build(:site)
+      assert_true(site.valid?)
+    end
+
+    test "blank" do
```

Comment:
Yes, It's intentional.
"blank" and "null" is special case because user cannot input these data via web UI.
But we should add test for these case, we should consider that "blank" and "null" may break something or not.